File: Design2P2.java
class MyHashSet {

    /** Initialize your data structure here. */
    int size = 10000;
     boolean[][] bucket;
    public MyHashSet() {
      bucket = new boolean[size][];
    }
    
    int hashCode(int key){
        return key%size;
    }
    
    public void add(int key) {
        int index = hashCode(key);
        if(bucket[index]==null)
            bucket[index] = new boolean[size+1];
        bucket[index][key/size] = true;
        
    }
    
    public void remove(int key) {
        int index = hashCode(key);
         if(bucket[index]==null)
             return;
         bucket[index][key/size] = false;
    }
    
    /** Returns true if this set contains the specified element */
    public boolean contains(int key) {
       int index = hashCode(key);
        if(bucket[index]==null)
            return false;
        else
            if(bucket[index][key/size])
                return true;
        return false;
    }
}

/**
 * Your MyHashSet object will be instantiated and called as such:
 * MyHashSet obj = new MyHashSet();
 * obj.add(key);
 * obj.remove(key);
 * boolean param_3 = obj.contains(key);
 */

File: Design-2-P-2.py
class MyHashSet:

    def __init__(self):
        """
        Initialize your data structure here.
        """
        self.hashset = [-1] * (10000000 - 1)

    def add(self, key) :

        self.hashset[key] = key

    def remove(self, key) :

        self.hashset[key] = -1

    def contains(self, key) :
        """
        Returns true if this set contains the specified element
        """
        if self.hashset[key] != -1:
            print "True"
        else:
            print "False"

# Your MyHashSet object will be instantiated and called as such:
obj = MyHashSet()
obj.add(1)
obj.add(2)
obj.remove(1)
obj.contains(2)
